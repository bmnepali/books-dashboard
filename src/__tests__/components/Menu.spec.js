import * as React from 'react';

// Load Component
import Menu from '../../components/Menu';

describe('Menu Component', () => {
  describe('rendering testing', () => {
    it('renders Menu component without any error', () => {
      shallow(<Menu />);
    });

    it('matches snapshot', () => {
      const wrapper = shallow(<Menu />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
