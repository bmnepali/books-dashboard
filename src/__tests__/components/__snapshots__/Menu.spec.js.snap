// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Component rendering testing matches snapshot 1`] = `
<section>
  <h3
    className="sidebar-heading"
  >
    Menu
  </h3>
  <ul
    className="nav flex-column"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link active"
        href="#"
      >
        <i
          className="icon ion-md-bookmarks"
        />
        My Books 
        <span
          className="sr-only"
        >
          (current)
        </span>
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="#"
      >
        <i
          className="icon ion-md-heart-empty"
        />
        Favorite Books
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="#"
      >
        <i
          className="icon ion-md-list"
        />
        My Categories
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="#"
      >
        <i
          className="icon ion-md-chatboxes"
        />
        My Messages
      </a>
    </li>
  </ul>
</section>
`;
